examples/coap-server.c

```diff
@@ -172,6 +177,39 @@ handle_sigint(int signum UNUSED_PARAM) {
   quit = 1;
 }
 
+static transient_value_t *
+alloc_resource_data(coap_binary_t *value) {
+  transient_value_t *transient_value;
+  if (!value)
+    return NULL;
+  transient_value = coap_malloc(sizeof(transient_value_t));
+  if (!transient_value) {
+    coap_delete_binary(value);
+    return NULL;
+  }
+  transient_value->ref_cnt = 1;
+  transient_value->value = value;
+  return transient_value;
+}
+
+/*
+ * Need to handle race conditions of data being updated (by PUT) and
+ * being read by a blocked response to GET.
+ */
+static void
+release_resource_data(coap_session_t *session UNUSED_PARAM,
+                      void *app_ptr) {
+  transient_value_t *value = (transient_value_t *)app_ptr;
```

Comment:
To improve readability, I suggest calling the local variable `transient_value` (in alignment with `alloc_resource_data()`).

Reply:
Updated